plugins/admission/pkg/admission/validate.go

```diff
@@ -135,20 +160,24 @@ func (v *Validator) Handle(ctx context.Context, req admission.Request) admission
 	return response
 }
 
-func getRequestReport(req admission.Request) (models.ReportInfo, error) {
-	report := models.ReportInfo{
-		Report:  "metadata",
-		Version: "0.1.0",
-	}
-	var err error
-	report.Contents, err = json.Marshal(&req.AdmissionRequest)
-	return report, err
+type MetadataReport struct {
+	admissionv1.AdmissionRequest
+	NamespaceObject map[string]any `json:"namespaceObject,omitempty"`
```

Comment:
Any suggestion on the name here? `namespace` is already taken by the namespace string value in `admissionv1.AdmissionRequest`

Reply:
A field name of `NamespaceMetadata` might be slightly more informative, but I don't feel strongly about the suggestion.

Reply:
Thanks, Ivan... I like `NamespaceMetadata` better, will change!

Reply:
Done